src/docs/content.go

```diff
@@ -508,13 +508,13 @@ Enterprise Performance Computing (EPC)`
       shub://user/image:tag`
 	PullExample string = `
   From Sylabs cloud library
-  $ singularity pull library://alpine:latest
+  $ singularity pull alpine.sif library://alpine:latest
 
   From Docker
-  $ singularity pull docker://tensorflow/tensorflow:latest
+  $ singularity pull tensorflow.sif docker://tensorflow/tensorflow:latest
 
   From Shub
-  $ singularity pull shub://vsoch/singularity-images`
+  $ singularity pull singularity-images.sif shub://vsoch/singularity-images`
```

Comment:
There is a '`' at the end that should not be there.

Reply:
The '\`' is the end of a multiline string. Compiling singularity without the '\`' will fail.